File: atcoder.jp/contests/code-festival-2016-qualb/tasks/codefestival_2016_qualB_a/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const S = input();
const T = "CODEFESTIVAL2016";

let ans = 0;
for (let i = 0; i < S.length; i++) if (S[i] != T[i]) ans++;

console.log(ans);

File: atcoder.jp/contests/code-festival-2016-qualb/tasks/codefestival_2016_qualB_a/main.rs
use proconio::input;

fn main() {
    input! {
        s: String,
    }

    let ans = s
        .chars()
        .zip("CODEFESTIVAL2016".chars())
        .fold(0, |acc, (ch1, ch2)| acc + (ch1 != ch2) as i32);

    println!("{}", ans);
}
